django/conf/project_template/project_name/urls.py

```diff
@@ -2,9 +2,15 @@
 from django.contrib import admin
 
 urlpatterns = [
-    # Examples:
-    # url(r'^$', '{{ project_name }}.views.home', name='home'),
-    # url(r'^blog/', include('blog.urls')),
+    # Example 1: Function Views
+    #   Step 1. Import view above:  from {{ project_name }}.views import home
+    #   Step 2. In this list:       url(r'^$', home, name='home'),
+    # Example 2: Class-Based Views
+    #   Step 1. Import view above:  from {{ project_name }}.views import Home
+    #   Step 2. In this list:       url(r'^$', Home.as_view(), name='home'),
+    # Example 3: Include URL Configuration
+    #   Step 1. Import above:       from blog import urls as blog_urls
+    #   Step 2. In this list:       url(r'^blog/', include(blog_urls)),
```

Comment:
True. However, I was under the impression that avoiding the namespace string was considered best practice, and opted to provide that instead. Am I mistaken?